flake.crane.nix

```diff
@@ -2,8 +2,9 @@
 craneLib:
 let
   # filter source code at path `src` to include only the list of `modules`
-  filterModules = modules: src:
+  filterModules = modules: raw-src:
     let
+      src = builtins.path { path = raw-src; name = "src"; };
```

Comment:
Any reason for the string "src" instead of e.g. "fedimint"? What would make the nox path names a bit easier to distinguish.

Reply:
No. I've thought it doesn't matter. Will switch to fedimint, I think it's better indeed.

Reply:
https://github.com/fedimint/fedimint/pull/1960